layouts/partials/sign-up-node-operator-email/form.html

```diff
@@ -44,7 +44,7 @@ <h2 class="light-black">What’s your email?</h2>
                 errElm.setAttribute("class", "row");
                 return;
             }
-            email = input.value
+            email = input.value.toLowerCase().trim()
```

Comment:
Do not modify the email that the user supplies us. They wrote the way they prefer it. It also doesn't do the right thing with unicode. The trim part is probably fine.